addons/default/visiosoft/defaultadmin-theme/resources/views/layouts/default.twig

```diff
@@ -24,6 +24,10 @@
 
         {{ breadcrumb() }}
 
+        {% if trans('module::addon.info') != 'module::addon.info' %}
+            <div class="page-info m-2 small text-white alert rounded">{{ trans('module::addon.info') }}</div>
+        {% endif %}
```

Comment:
you can use the example found here /admin/system/dumps you don't need to create a new div